server/src/main/java/org/eclipse/lsp/cobol/core/preprocessor/delegates/GrammarPreprocessorListenerImpl.java

```diff
@@ -340,14 +345,24 @@ private CopybookModel getCopyBookContent(String copybookName, Locality locality)
     CopybookModel copybook =
         copybookService.resolve(copybookName, documentUri, copybookProcessingMode);
 
-    if (copybook.getContent() == null) {
+    if (copybook.getContent() == null && !isCobybookImplictlyDefined(copybookName)) {
```

Comment:
Please, rename and invert the logic of `isCobybookImplictlyDefined` to avoid the negation of its call.